app/views/pets/show.html.erb

```diff
@@ -7,15 +7,17 @@
 <p>Availability: <%= @pet.adopted? %></p>
 <% if  @pet.adopted? == "Pending" %>
   <p> On hold for <%= "#{@pet.applications[0].name}" %> </p>
+<% else %>
+  <%= link_to "Delete #{@pet.name}", "/shelters/#{@pet.shelter.id}/pets/#{@pet.id}", method: :delete %> /
```

Comment:
`delete` button is only available when the `pet` has not been adopted yet (ie. when its application has not been approved yet)